### ISSUE TITLE
news_channel error 400 bad_request

### ISSUE THREAD

**Issue by LucaMRe3** [Lucky3028/terraform-provider-discord#149]
I implemented the creation of a news channel on my terraform template but when I try to create even a simple new channel with only name, and discord_id specified I get this error.
│ Error: Failed to create channel: HTTP 400 Bad Request, {"message": "Invalid Form Body", "code": 50035, "errors": {"type": {"_errors": [{"code": "BASE_TYPE_CHOICES", "message": "Value must be one of {0, 2, 4, 6, 13, 14, 15, 16}."}]}}}
│
│   with discord_news_channel.sos,
│   on main.tf line 51, in resource "discord_news_channel" "sos":
│   51:  resource "discord_news_channel" "sos" {
│

this is the resource code:

resource "discord_news_channel" "sos" {
   name = "sos"
   server_id = var.Server_id_value
 }

**Comment by Lucky3028** [comment]
I have not tried yet, but that is strange and may be discordgo's bug. I want to check it later, but I'll have been busy for about 2 weeks.

**Comment by LucaMRe3** [comment]
sure, thanks for the support. I noticed that there is the bug since the 1.6.1 version of the provider

**Comment by Lucky3028** [comment]
> I noticed that there is the bug since the 1.6.1 version of the provider

I see. That's very helpful information. If that's so, I found it's not discordgo's, but this provider's.

**Comment by LucaMRe3** [comment]
Yeah, but I didn't test the other version, I was meaning that I have tested since the 1.6.1. Sorry if I explaned it wrong

**Comment by ktriponis** [comment]
This looks like a server configuration issue rather than a provider bug. Discord only allows creating announcement channels (type 5) in servers that have the `COMMUNITY` feature enabled — which grants the `NEWS` feature.

The error `"Value must be one of {0, 2, 4, 6, 13, 14, 15, 16}"` is Discord's API rejecting type 5 because the server doesn't support it.

To fix: enable Community in your server settings (Server Settings → Enable Community), then the announcement channel creation should work.

**Comment by Lucky3028** [comment]
Thanks for explanation. This behavior should be represented in some documents of this provider.
